scripts/package.cjs

```diff
@@ -18,10 +18,8 @@ rootPackage.exports = {
   '.': {
     types: './index.d.ts',
     svelte: './index.js',
-    default: './index.js',
+    style: "./styles/index.css"
   },
-  './tailwind.config': './tailwind.config.js',
-  './tailwindColors': './styles/tailwindColors.js',
   './styles': './styles/index.css',
 };
 
```

Comment:
I think instead of the existing rootPackage.style, we just want `rootPackage.style = 'styles/index.css';`